messenger.rst

```diff
@@ -623,6 +623,32 @@ config and start your workers:
 
 See the `Supervisor docs`_ for more details.
 
+
+.. tip::
+
+    To get a deterministic worker name like for e.g. the Redis transport, you may pass the process number as an
```

Comment:
I miss information **why** I would like to do this. 

We do this to leverage the Redis stream "consumer group" feature, right?

Reply:
Correct. To me the **why** doesn't really matter here. It's just a tip and as we're in the Subervisor section here I don't want to get specific about Redis. I'm sure there are other use cases where you may want to set an environment variable.